### PR TITLE
Feature/local server2boosterdocs

### PR DESCRIPTION
**Pull request by adriangmweb**
There's the local server acting as a proxy between the extension and the Booster Doc API! 🚀 

image

Next step will be starting the server automatically when the extension starts.

### CODE REVIEW

File: apps/server/src/server.ts [hunk 1]
```diff
@@ -6,10 +6,34 @@ server.get("/ping", async () => {
   return "pong\n";
```
Comment (NickSeagull) [inline apps/server/src/server.ts]: Maybe we can delete the pong endpoint?

File: apps/server/src/server.ts [hunk 2]
```diff
@@ -6,10 +6,34 @@ server.get("/ping", async () => {
...
+      if (!answer.ok) {
+        throw new Error(`Error from API: ${answer.statusText}`);
+      }
+
+      return answer;
+    } catch (error) {
+      request.log.error(error);
+      return response
+        .code(500)
+        .send({ error: "There has been an error processing your request" });
```
Comment (NickSeagull) [inline apps/server/src/server.ts]: Maybe we can return a more human response so the chat gives the impression of "talking"?

**Review by NickSeagull — APPROVED** [review]
Made two comments, looking good!

File: apps/server/src/server.ts (hunk 1 above)
Comment (adriangmweb) [reply apps/server/src/server.ts]: I left it there so we could debug while working locally, but it makes sense!

**Pull request merged**